docs/developer-docs/backend/periodic-tasks.md

```diff
@@ -53,6 +53,18 @@ Despite its superiority over the heartbeats, the CDK timers library has a few kn
    * As the canister output queue is limited in size (500 messages at the moment), this implicitly limits the number of tasks which might be scheduled in one round.
 3. **Advanced scheduling.** The CDK timers library uses relative time to schedule tasks. To use an absolute time, canister developers should calculate the duration between now and the point in time, or use a third party library.
 
+Frequently Asked Questions
+--------------------------
+
+Q: Do timers support Deterministic Time Slicing (DTS)? \
+A: Yes, as the CDK timers library initiates a self canister call to execute each task, normal [Update Message instruction limit](../production/instruction-limits.md) apply with DTS enabled.
```

Comment:
Formatting looks weird for all questions: 
<img width="294" alt="image" src="https://user-images.githubusercontent.com/14757975/225327312-874ab29e-9ba9-48fc-86ec-91638a073741.png">